langchain/evaluation/run_evaluators/config.py

```diff
@@ -0,0 +1,78 @@
+"""Configuration for run evaluators."""
+from typing import Any, Dict, List, Optional
+
+from langsmith import RunEvaluator
+from pydantic import BaseModel
+
+from langchain.embeddings.base import Embeddings
+from langchain.evaluation.criteria.eval_chain import CRITERIA_TYPE
+from langchain.evaluation.embedding_distance.base import EmbeddingDistance
+from langchain.evaluation.schema import EvaluatorType
+from langchain.evaluation.string_distance.base import StringDistance
+from langchain.schema.language_model import BaseLanguageModel
+from langchain.schema.prompt_template import BasePromptTemplate
+
+
+class EvalConfig(BaseModel):
+    """Configuration for a given run evaluator."""
+
+    evaluator_type: EvaluatorType
+    """The type of evaluator to use."""
+
+    def get_kwargs(self) -> Dict[str, Any]:
+        return self.dict(exclude={"evaluator_type"}, exclude_none=True)
+
+
+class RunEvalConfig(BaseModel):
+    """Configuration for a run evaluation."""
+
+    evaluator_configs: List[EvalConfig]
+    # TODO: Should support custom langchain evaluators here too?
+    custom_run_evaluators: Optional[List[RunEvaluator]] = None
```

Comment:
Could just make `custom_evaluators` and accept any run evaluator or `Union[StringEvaluator, <other eval interface types>]` for custom evals that aren't loadable